std_msgs/Time stamp
float64 x
float64 x_dot